### DISCUSSION TITLE
does anyone can tell me what pugi stand for?

### DISCUSSION THREAD

**Discussion by bibaodi** [zeux/pugixml#542] [Q&A] [answered]
I using bing.com searched a lot, and read the readme related documents still failed to know the name's meaning.
does anyone can tell me what pugi stand for?
Thanks a lot.

**Comment by zeux** [comment, marked as answer]
`pug` stands for `pugnacious` and comes from `pugxml`, `i` stands for `improved`.

**Reply by bibaodi** [reply]
Very Thanks ~
